spec/spec_helper.rb

```diff
@@ -1,5 +1,4 @@
-require 'simplecov'
-SimpleCov.minimum_coverage 95
-SimpleCov.start
+require 'codeclimate-test-reporter'
```

Comment:
rubocop is set the other way around: prefer single quotes when there's no interpolation